hack/chains/deploy_local_rekor.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+#
+# A wrapper for deploying rekor into a local cluster.
+# Uses the deploy_local_rekor.tpl file, which has a placeholder for
+# {domain} which is replaced by getting the domain from the openshift
+# cluster. 
+
+# Make sure we have our template file
+if [[ ! -f $PWD/deploy_local_rekor.tpl ]];then
+  echo "Template file 'deploy_local_rekor.tpl' not found in this directory "; exit 1 
+fi
+
+# Test if we have the argocd cli command:
+if ! command -v argocd &> /dev/null; then
+  echo "The argocd CLI is not installed. Please install and try again."
+  echo "See https://argo-cd.readthedocs.io/en/stable/getting_started/#2-download-argo-cd-cli for details"
+  exit 1
+fi
+
+# Get our domain from the ingresses config of the cluster
+domain=$( kubectl get ingresses.config.openshift.io cluster -o json|jq -r ".spec.domain" )
+
+# Set the template input file
+input_file="$PWD/deploy_local_rekor.tpl"
+
+output=$( sed -e "s/\${domain}/$domain/" $input_file )
+
+# If we want to see what's being written to the deployment file
+if [[ $1 == "--verbose" ]]; then
+  echo "Here's our rekor-server configuration yaml"
+  echo ""
+  echo "--------"
+  echo "$output" | yq e -P - 
+  echo "--------"
+fi
+
+# Ensure we're actually logged in before trying to execute argocd command
+
+# Find the list of argo cd apps to test logged in status
+ARGO_APPS=$( argocd app list -o name )
+
+if [[ $? != 0 ]]; then
+  # Assume the user is not logged in.
+  # Try to be helpful.
+
+  ARGO_HOST=$(
+    kubectl get route/openshift-gitops-server -n openshift-gitops \
+      -o jsonpath='{.spec.host}' )
+
+  LOGIN_CMD="argocd login $ARGO_HOST --sso --grpc-web"
+
+  echo
+  echo "*** Please log in with argocd and try again ***"
+  echo
+  echo "Running login command:"
+  echo "  $LOGIN_CMD"
+  echo
+  $LOGIN_CMD
+  echo
+  exit 1
+fi
+
+#argocd app create --grpc-web --upsert -f $deployment_file #2> /dev/null
+echo $output | argocd app create --grpc-web --upsert -f - 
```

Comment:
```suggestion
echo "$output" | argocd app create --grpc-web --upsert -f - 
```

I needed to add this on my end as `$output` contains newlines